components/function-controller/config/config.yaml

```diff
@@ -32,47 +32,47 @@ items:
   kind: ConfigMap
   metadata:
     labels:
-      function: runtime-function-controller
+      function: serverless-function-controller
     name: dockerfile-nodejs-8
-    namespace: runtime-system
+    namespace: serverless-system
 - apiVersion: v1
   data:
-    dockerRegistry: ### put your github name here e.g. k15r
+    dockerRegistry: 
     runtimes: |
       - ID: nodejs8
         dockerFileName: dockerfile-nodejs-8
       - ID: nodejs6
         dockerFileName: dockerfile-nodejs-6
-    serviceAccountName: runtime-controller
+    serviceAccountName: serverless-controller
   kind: ConfigMap
   metadata:
     labels:
       app: kubeless
     name: fn-config
-    namespace: runtime-system
+    namespace: serverless-system
 kind: List
 ---
 apiVersion: v1
 kind: Secret
 metadata:
   name: docker-reg-credential
-  namespace: runtime-system
+  namespace: serverless-system
   annotations:
     build.knative.dev/docker-0: https://index.docker.io/v1/
 type: kubernetes.io/basic-auth
 data:
   # your docker.io username goes here
   # username NOT the email
   # echo -n $USERNAME | base64
-  username:
+  username: 
   # password for the account
   # echo -n $PASSWORD | base64
-  password:
+  password: 
 ---
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: runtime-controller
-  namespace: runtime-system
+  name: serverless-controller
+  namespace: serverless-system
```

Comment:
```suggestion
  labels:
      app: serverless-controller
  namespace: serverless-system
```

Reply:
Not sure about that naming, we'd have a "controller" that is part of another "controller". Why not using a more generic naming such as "kyma serverless"?

Reply:
as discussed `function-controller` stays :)